### PR TITLE
STREAMS-1597 Restructure Install

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Broke out Install topics:
Install (index)
Helm parameters
Upgrade
Uninstall
Backup and disaster recovery
Note: I did not address the heading issue in the install topic. I will do that next.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links

### CODE REVIEW

**Comment by lbadenhop** [comment]
@allandenis38 Ready for your review

**Comment by guillaumelamirand** [comment]
Just a comment your branch has been created from develop, so you should apply the PR over develop branch instead of master.

**Comment by allandenis38** [comment]
@lbadenhop FYI as mentioned by @guillaumelamirand previous comment I changed the target branch to develop.

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@lbadenhop I am fine with the proposed restructuring of the install section.

**Comment by lbadenhop** [comment]
@allandenis38 Ready for your review. Also fixed broken link on index page. I cannot validate the link to "contribution guidelines" because Netlify for Streams seems to be unavailable.

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -438,18 +441,18 @@ export PEM_PATH="my-pem-path"
 kubectl create secret generic "${SECRET_NAME}" -n "${NAMESPACE}" --from-file="${PEM_PATH}" [--from-file=<other-pem-path>]
 ```
 
-* Set the [Helm parameters](#helm-parameters) `streams.extraCertificatesSecrets` to your `$SECRET_NAME`. If you have more than one secrets, they must be separated by a comma.
+* Set the [Helm parameters](/docs/install/helm-parameters/) `streams.extraCertificatesSecrets` to your `$SECRET_NAME`. If you have more than one secrets, they must be separated by a comma.
 
-### Monitoring
+## Monitoring
 
 Streams ships with monitoring. You can activate metrics with the parameters listed in [Monitoring parameters](#monitoring-parameters),
```
Comment (allandenis38) [inline content/en/docs/install/_index.md:448]:
```suggestion
Streams ships with monitoring. You can activate metrics with the parameters listed in [Monitoring parameters](/docs/install/helm-parameters/#monitoring-parameters),
```

**Review by allandenis38 — CHANGES_REQUESTED** [review]

**Review by allandenis38 — APPROVED** [review]

**Pull request merged**